src/styles/clock.scss

```diff
@@ -0,0 +1,59 @@
+@import "./variables.scss";
+
+.clock {
+  width: $clock-size;
+  height: $clock-size;
+  border: 5px dotted black;
+  border-radius: 50%;
+
+  &__hand {
+    position: absolute;
+    z-index: 2;
+
+    &--minutes {
+      top: 10%;
+      margin-left: 50%;
+      transform: translateX(-50%);
```

Comment:
I think the above 3 properties are duplicated for minutes and hours hands, so pls move them to __hand block)

Reply:
done